website/homepage/src/guides/__init__.py

```diff
@@ -174,10 +162,6 @@ def build_gallery(output_dir, jinja_env):
     output_file = os.path.join(output_folder, "index.html")
     with open(output_file, "w") as index_html:
         index_html.write(output)
-    output_file = os.path.join(output_dir, "guides.html")
-    with open(output_file, "w") as index_html:
-        index_html.write(output)
```

Comment:
Same, what was this for?

Reply:
Same reason as above, we had launched guides with a link to `/guides.html` but then switched to `/guides` so I had created both to be supported. but now I redirect them in nginx.